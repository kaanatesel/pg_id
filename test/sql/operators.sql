SELECT '507f1f77bcf86cd799439011'::mgid = '507f1f77bcf86cd799439011'::mgid;
SELECT '507f1f77bcf86cd799439011'::mgid = '507f1f77bcf86cd799439222'::mgid;

SELECT '507f1f77bcf86cd799439011'::mgid <> '507f1f77bcf86cd799439222'::mgid;
SELECT '507f1f77bcf86cd799439011'::mgid <> '507f1f77bcf86cd799439011'::mgid;

SELECT '507f1f77bcf86cd799439011'::mgid < '507f1f77bcf86cd799439011'::mgid;
SELECT '507f1f77bcf86cd799439011'::mgid < '507f1f77bcf86cd799439222'::mgid;
SELECT '507f1f77bcf86cd799439222'::mgid < '507f1f77bcf86cd799439011'::mgid;

SELECT '507f1f77bcf86cd799439011'::mgid > '507f1f77bcf86cd799439011'::mgid;
SELECT '507f1f77bcf86cd799439011'::mgid > '507f1f77bcf86cd799439222'::mgid;
SELECT '507f1f77bcf86cd799439222'::mgid > '507f1f77bcf86cd799439011'::mgid;

SELECT '507f1f77bcf86cd799439011'::mgid <= '507f1f77bcf86cd799439011'::mgid;
SELECT '507f1f77bcf86cd799439011'::mgid <= '507f1f77bcf86cd799439222'::mgid;
SELECT '507f1f77bcf86cd799439222'::mgid <= '507f1f77bcf86cd799439011'::mgid;

SELECT '507f1f77bcf86cd799439011'::mgid >= '507f1f77bcf86cd799439011'::mgid;
SELECT '507f1f77bcf86cd799439011'::mgid >= '507f1f77bcf86cd799439222'::mgid;
SELECT '507f1f77bcf86cd799439222'::mgid >= '507f1f77bcf86cd799439011'::mgid;


SELECT '507f1f77bcf86cd799439011'::mgid = '507f1f77bcf86cd799439011';
SELECT '507f1f77bcf86cd799439011'::mgid = '507f1f77bcf86cd799439222';

SELECT '507f1f77bcf86cd799439011'::mgid <> '507f1f77bcf86cd799439222';
SELECT '507f1f77bcf86cd799439011'::mgid <> '507f1f77bcf86cd799439011';

SELECT '507f1f77bcf86cd799439011'::mgid < '507f1f77bcf86cd799439011';
SELECT '507f1f77bcf86cd799439011'::mgid < '507f1f77bcf86cd799439222';
SELECT '507f1f77bcf86cd799439222'::mgid < '507f1f77bcf86cd799439011';

SELECT '507f1f77bcf86cd799439011'::mgid > '507f1f77bcf86cd799439011';
SELECT '507f1f77bcf86cd799439011'::mgid > '507f1f77bcf86cd799439222';
SELECT '507f1f77bcf86cd799439222'::mgid > '507f1f77bcf86cd799439011';

SELECT '507f1f77bcf86cd799439011'::mgid <= '507f1f77bcf86cd799439011';
SELECT '507f1f77bcf86cd799439011'::mgid <= '507f1f77bcf86cd799439222';
SELECT '507f1f77bcf86cd799439222'::mgid <= '507f1f77bcf86cd799439011';

SELECT '507f1f77bcf86cd799439011'::mgid >= '507f1f77bcf86cd799439011';
SELECT '507f1f77bcf86cd799439011'::mgid >= '507f1f77bcf86cd799439222';
SELECT '507f1f77bcf86cd799439222'::mgid >= '507f1f77bcf86cd799439011';


SELECT '507f1f77bcf86cd799439011' = '507f1f77bcf86cd799439011'::mgid;
SELECT '507f1f77bcf86cd799439011' = '507f1f77bcf86cd799439222'::mgid;

SELECT '507f1f77bcf86cd799439011' <> '507f1f77bcf86cd799439222'::mgid;
SELECT '507f1f77bcf86cd799439011' <> '507f1f77bcf86cd799439011'::mgid;

SELECT '507f1f77bcf86cd799439011' < '507f1f77bcf86cd799439011'::mgid;
SELECT '507f1f77bcf86cd799439011' < '507f1f77bcf86cd799439222'::mgid;
SELECT '507f1f77bcf86cd799439222' < '507f1f77bcf86cd799439011'::mgid;

SELECT '507f1f77bcf86cd799439011' > '507f1f77bcf86cd799439011'::mgid;
SELECT '507f1f77bcf86cd799439011' > '507f1f77bcf86cd799439222'::mgid;
SELECT '507f1f77bcf86cd799439222' > '507f1f77bcf86cd799439011'::mgid;

SELECT '507f1f77bcf86cd799439011' <= '507f1f77bcf86cd799439011'::mgid;
SELECT '507f1f77bcf86cd799439011' <= '507f1f77bcf86cd799439222'::mgid;
SELECT '507f1f77bcf86cd799439222' <= '507f1f77bcf86cd799439011'::mgid;

SELECT '507f1f77bcf86cd799439011' >= '507f1f77bcf86cd799439011'::mgid;
SELECT '507f1f77bcf86cd799439011' >= '507f1f77bcf86cd799439222'::mgid;
SELECT '507f1f77bcf86cd799439222' >= '507f1f77bcf86cd799439011'::mgid;