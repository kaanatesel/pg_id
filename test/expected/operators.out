SELECT '507f1f77bcf86cd799439011'::mgid = '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid = '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid <> '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid <> '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid < '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid < '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439222'::mgid < '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid > '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid > '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439222'::mgid > '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid <= '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid <= '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439222'::mgid <= '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid >= '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid >= '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439222'::mgid >= '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid = '507f1f77bcf86cd799439011';
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid = '507f1f77bcf86cd799439222';
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid <> '507f1f77bcf86cd799439222';
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid <> '507f1f77bcf86cd799439011';
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid < '507f1f77bcf86cd799439011';
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid < '507f1f77bcf86cd799439222';
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439222'::mgid < '507f1f77bcf86cd799439011';
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid > '507f1f77bcf86cd799439011';
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid > '507f1f77bcf86cd799439222';
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439222'::mgid > '507f1f77bcf86cd799439011';
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid <= '507f1f77bcf86cd799439011';
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid <= '507f1f77bcf86cd799439222';
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439222'::mgid <= '507f1f77bcf86cd799439011';
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid >= '507f1f77bcf86cd799439011';
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011'::mgid >= '507f1f77bcf86cd799439222';
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439222'::mgid >= '507f1f77bcf86cd799439011';
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011' = '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011' = '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011' <> '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011' <> '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011' < '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011' < '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439222' < '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011' > '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011' > '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439222' > '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011' <= '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011' <= '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439222' <= '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439011' >= '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 t
(1 row)

SELECT '507f1f77bcf86cd799439011' >= '507f1f77bcf86cd799439222'::mgid;
 ?column? 
----------
 f
(1 row)

SELECT '507f1f77bcf86cd799439222' >= '507f1f77bcf86cd799439011'::mgid;
 ?column? 
----------
 t
(1 row)

